DeepSpeech/Dockerfile.train

```diff
@@ -3,6 +3,7 @@ FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu18.04
 ARG ds_repo=mozilla/DeepSpeech
 ARG ds_branch=92d8bad7c113885690867a6128a7728325f540f3
 ARG ds_sha1=92d8bad7c113885690867a6128a7728325f540f3
+ARG cc_repo=MestafaKamal/CorporaCreator
```

Comment:
We should not use your fork but rather the official repo, and pin to a specific sha1 like it is done for the others